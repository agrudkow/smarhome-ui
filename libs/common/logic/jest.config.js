module.exports = {
  name: 'common-logic',
  preset: '../../../jest.config.js',
  transform: {
    '^.+\\.[tj]sx?$': 'ts-jest',
  },
  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
  coverageDirectory: '../../../coverage/libs/common/logic',
};
