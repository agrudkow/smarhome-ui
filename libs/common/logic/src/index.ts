export * from './lib/use-accordion';
export * from './lib/use-window-dimensions';
