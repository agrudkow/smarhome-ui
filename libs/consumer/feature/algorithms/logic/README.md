# consumer-feature-algorithms-logic

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `nx test consumer-feature-algorithms-logic` to execute the unit tests via [Jest](https://jestjs.io).
