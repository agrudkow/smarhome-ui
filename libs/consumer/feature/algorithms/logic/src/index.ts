export * from './lib/algorithm.interface';
export * from './lib/algorithms-cells-parser';
export * from './lib/algorithms-data-parser';
export * from './lib/use-algorithms-list';
export * from './lib/use-algorithm-detials';
export * from './lib/use-algorithm-rating';
export * from './lib/use-select-algorithm';
