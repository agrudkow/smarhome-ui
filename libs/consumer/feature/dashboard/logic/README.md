# consumer-feature-dashboard-logic

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `nx test consumer-feature-dashboard-logic` to execute the unit tests via [Jest](https://jestjs.io).
