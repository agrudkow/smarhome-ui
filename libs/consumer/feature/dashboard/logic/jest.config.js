module.exports = {
  name: 'consumer-feature-dashboard-logic',
  preset: '../../../../../jest.config.js',
  transform: {
    '^.+\\.[tj]sx?$': 'ts-jest',
  },
  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
  coverageDirectory:
    '../../../../../coverage/libs/consumer/feature/dashboard/logic',
};
