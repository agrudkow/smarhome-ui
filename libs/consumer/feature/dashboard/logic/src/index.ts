export * from './lib/use-billings';
