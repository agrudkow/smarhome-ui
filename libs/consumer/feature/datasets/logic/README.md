# consumer-feature-datasets-logic

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `nx test consumer-feature-datasets-logic` to execute the unit tests via [Jest](https://jestjs.io).
