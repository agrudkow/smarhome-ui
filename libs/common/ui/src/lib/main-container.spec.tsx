import React from 'react';
import { customRender } from '@smarthome/common/logic';
import MainContainer from './main-container';

describe(' MainContainer', () => {
  it('should render successfully', () => {
    const { baseElement } = customRender(
      <MainContainer dissableScroll={false} />
    );
    expect(baseElement).toBeTruthy();
  });

  it('should match snapshot', () => {
    const { baseElement } = customRender(
      <MainContainer dissableScroll={false} />
    );
    expect(baseElement.firstChild).toMatchSnapshot();
  });
});
