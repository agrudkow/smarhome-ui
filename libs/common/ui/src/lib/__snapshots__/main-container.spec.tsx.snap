// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` MainContainer should match snapshot 1`] = `
.c0 {
  min-height: 100%;
  margin: 0;
  padding: 5px;
  overflow: unset;
  height: unset;
}

@media (min-width:769px) {
  .c0 {
    padding: 10px;
    overflow: unset;
    height: unset;
  }
}

@media (min-width:1024px) {
  .c0 {
    padding: 20px;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
